rockcraft/project.py

```diff
@@ -242,7 +269,7 @@ def _validate_license(cls, rock_license: str) -> str:
     def _validate_title(cls, title: Optional[str], values: Mapping[str, Any]) -> str:
         """If title is not provided, it defaults to the provided ROCK name."""
         if not title:
-            title = values["name"]
+            title = values.get("name", "")
```

Comment:
Why does this validatory need to run `always`? Since `name` is mandatory, this `.get` looks more of a workaround to avoid a false positive exception...

Reply:
title can be validated before name